rskj-core/src/test/java/co/rsk/peg/BridgeSupportRegisterBtcTransactionTest.java

```diff
@@ -91,8 +94,8 @@ class BridgeSupportRegisterBtcTransactionTest {
     private BtcLockSenderProvider btcLockSenderProvider;
     private PeginInstructionsProvider peginInstructionsProvider;
 
-    private List<UTXO> retiringFederationUtxos = new ArrayList<>();
-    private List<UTXO> activeFederationUtxos = new ArrayList<>();
+    private final List<UTXO> retiringFederationUtxos = new ArrayList<>();
+    private final List<UTXO> activeFederationUtxos = new ArrayList<>();
```

Comment:
I think these two fields should be instantiated in the setup method since every test using it possibly adds elements to the instance list.

Reply:
Just trying to fix the tests to compile at this stage :) Don't want to get into too much refactoring or it will be a never ending story